### DISCUSSION TITLE
MariaDB

### DISCUSSION THREAD

**Discussion by robertsilen** [jeremyevans/sequel#2170] [General]
Any reason not to mention MariaDB in the [README.rdoc](https://github.com/jeremyevans/sequel/blob/master/README.rdoc)? Could it be added?

**Comment by jeremyevans** [comment]
The README lists adapters supported, not databases supported.  There is no MariaDB adapter.  The mysql, mysql2, trilogy, and JDBC adapters all work with MariaDB.

**Reply by robertsilen** [reply]
Maybe that could be mentioned in the README?

**Reply by jeremyevans** [reply]
I think most MariaDB users are aware it uses the same protocol as MySQL and database adapters that can connect to MySQL can also connect to MariaDB.

I don't want to clutter the README with this.  I would be open to adding something to doc/opening_databases.rdoc that lists databases that Sequel supports, and the adapters that can be used with each.
